### DISCUSSION TITLE
Data Import Stucked

### DISCUSSION THREAD

**Discussion by borisdagnon** [pimcore/pimcore#16237] [Q&A / Support]
Hi,
When trying to import data from an excel file, the progress bar is blocked here
![image](https://github.com/pimcore/pimcore/assets/16353265/772d7494-e4ef-4ec3-81df-ab958ca6d48e)

This si my configuration for the execution part
![image](https://github.com/pimcore/pimcore/assets/16353265/3d73a841-5316-4733-aab9-03b72311bbc9)

![image](https://github.com/pimcore/pimcore/assets/16353265/f7df19ba-c192-4efc-b5d3-aeae628af038)

**Comment by kingjia90** [comment]
Does it usually work or is this specific file/case? 
Did you set the process queue command cron? Please see https://pimcore.com/docs/platform/Data_Importer/Installation#command-based and then https://pimcore.com/docs/platform/Data_Importer/Import_Execution_Details , if it is still stuck, please provide further info for example bundle and pimcore versions.

**Reply by borisdagnon** [reply]
Hi,
Where do i type cron commands ?
Thanks.

**Reply by kingjia90** [reply]
In the server terminal, or configure supervisord

**Reply by borisdagnon** [reply]
Ok.
I got this:
![image](https://github.com/pimcore/pimcore/assets/16353265/7dc22a69-7134-4314-89a2-d7e1e2ea2928)
But then what do i have to do ?

**Reply by borisdagnon** [reply]
For information when i type the command with the stars  have this error:
![image](https://github.com/pimcore/pimcore/assets/16353265/d95cf107-036d-4e32-bdbf-06f089e93526)

**Reply by kingjia90** [reply]
is this file missing in the root project? 
https://github.com/pimcore/skeleton/blob/11.x/bin/console

**Reply by borisdagnon** [reply]
No it seems like it's present
![image](https://github.com/pimcore/pimcore/assets/16353265/f3b4fcb9-b94e-45ef-8b5a-a156edd29858)

**Reply by borisdagnon** [reply]
I put the cron jobs in the cron file
![image](https://github.com/pimcore/pimcore/assets/16353265/72fe2a1a-c179-40a4-9de6-e74a6c03b998)
But when it executes, it's always stuck
![image](https://github.com/pimcore/pimcore/assets/16353265/7cbb94a4-5388-4296-b3ad-04791be5628a)

**Reply by kingjia90** [reply]
is it checked as `Active` under the `General` tab?

**Reply by kingjia90** [reply]
suggesting to try to run `bin/console datahub:data-importer:process-queue-sequential` directly in the terminal (without cronjob, as i can't see if it's properly set and running)

**Reply by borisdagnon** [reply]
Hi,
Yes it is active:
![image](https://github.com/pimcore/pimcore/assets/16353265/9d7f37f6-2663-4824-a55c-4661e82d1612)

That is what i get when i try to import with your command:
![image](https://github.com/pimcore/pimcore/assets/16353265/2363d593-7026-49be-8914-9d39b15c7d86)

**Reply by kingjia90** [reply]
beside the errors, the progress bar should have advanced and completed the queue

**Reply by borisdagnon** [reply]
Yes it's the case
I now configured the cron with the sequencial import and it is running well.
Don't know why  when i set the Parallel config, it doesn't work

**Reply by kingjia90** [reply]
that's good to hear, at least it's not stucked.

It looks like regex validation `\{w{1,40}$` is set to not accept any symbols (with `w`), and in there you got `% RAYON 5% SPANDEX` and a `http://`

**Reply by kingjia90** [reply]
If the sequential import is broken and can provide the steps to reproduce it, please [report the issue ](https://github.com/pimcore/data-importer/issues/new?assignees=&labels=Bug&projects=&template=Bug-Report.yml&title=%5BBug%5D%3A+)
